### ISSUE TITLE
[vcpkg] Documentation messed up.

### ISSUE THREAD

**Issue by cnSchwarzer** [microsoft/vcpkg#29019]
Recently I'm using binary caching and I searched manuals. I found multiple vcpkg documentation sources and I'm confused about which one is the latest and most reliable. There are at least 5 sources:

- https://vcpkg.io
- https://vcpkg.readthedocs.io
- https://github.com/microsoft/vcpkg/docs
- https://learn.microsoft.com/vcpkg
- https://github.com/microsoft/vcpkg-docs

I think `vcpkg-docs` is the latest one. Because it contains docs about `x-cos` and other custom binary caching sources, but `vcpkg` and other websites point `latest` to itself

> The latest version of this documentation is available on [GitHub](https://github.com/Microsoft/vcpkg/tree/master/docs/README.md).

While `vcpkg-docs` is like an orphan and has no clear README in that repo.

In summary, the multiple sources situation is frustrating and should be corrected ASAP. Fix the link in vcpkg/Readme, redirect websites, and add Readme to vcpkg-docs

**Comment by BillyONeal** [comment]
https://learn.microsoft.com/vcpkg is intended to be the canonical location.

### https://github.com/microsoft/vcpkg-docs

This is the git repo powering the learn site. If you are on the learn site and click the pencil icon it takes you here:

<img width="719" alt="learn.microsoft.com example" src="https://user-images.githubusercontent.com/1544943/217075065-2849ab95-02f3-46cd-b9dc-2afca52358fa.png">

### https://github.com/microsoft/vcpkg/docs

This used to be the 'source of truth' from which all other docs were generated; I believe this will go away soon and all content will be moved to https://github.com/microsoft/vcpkg-docs

### https://vcpkg.io/

@JavierMatosD is working on changes to point this to learn. https://github.com/vcpkg/vcpkg.github.io/pull/93

### https://vcpkg.readthedocs.io/

@ras0219-msft says this should be fixed to be shutdown / redirect

**Comment by cnSchwarzer** [comment]
Thanks for the answer!

**Comment by MonicaLiu0311** [comment]
> https://vcpkg.readthedocs.io/
> @ras0219-msft says this should be fixed to be shutdown / redirect

How is the progress of this matter?
https://github.com/microsoft/vcpkg/issues/31059#issuecomment-1518977043

**Comment by cnSchwarzer** [comment]
@MonicaLiu0311 It seems already redirecting to https://learn.microsoft.com/vcpkg/ from my view

**Comment by cnSchwarzer** [comment]
It seems all situations are pointing to https://learn.microsoft.com/zh-cn/vcpkg/, closing this issue! Thanks!
